libhoney/src/main/java/io/honeycomb/libhoney/DefaultDebugResponseObserver.java

```diff
@@ -0,0 +1,54 @@
+package io.honeycomb.libhoney;
+
+import io.honeycomb.libhoney.responses.ClientRejected;
+import io.honeycomb.libhoney.responses.ServerAccepted;
+import io.honeycomb.libhoney.responses.ServerRejected;
+import io.honeycomb.libhoney.responses.Unknown;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.UnknownHostException;
+
+public class DefaultDebugResponseObserver implements ResponseObserver {
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultDebugResponseObserver.class);
+
+    protected static final String ERROR_TEMPLATE_401 = "Server responded with a 401 HTTP error code to a batch request." +
+        " This is likely caused by using an incorrect 'Team Write Key'. Check https://ui.honeycomb.io/account to verify your " +
+        "team write key. Rejected event: {}";
+    protected static final String ERROR_TEMPLATE_UNKNOWN_HOST_EXCEPTION = "Could not reach determine destination server. " +
+        "This could be due to 1) invalid apiHost 2) Could not resolve apiHost (firewall? dns?) 3) problem with proxy configuration. " +
+        "Unknown event: {}";
```

Comment:
Not sure the word "determine" belongs here? Could we rewrite as "Could not reach destination server...". I think that's clearer.

Reply:
Agreed and removed.